assignments/arrays.js

```diff
@@ -63,34 +63,67 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+function search(id) {
+    for (let i = 0; i < inventory.length; i++) {
+        if(id === inventory[i].id) {
+            console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`)
+        }
+    }
+}
+console.log(search(33));
```

Comment:
Again, this `console.log()` is actually logging `undefined` to your console after it runs the `search(33)` method, which logs the correct string. This is because you don't have a `return` in your `search()` method.

Here is what I would have done:
```
function search(id) {
    for (let i = 0; i < inventory.length; i++) {
        if(id === inventory[i].id) {
            return `Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`
        }
    }
}
```